### PR TITLE
chore: Add docs for ssh tunneling

### PR DESCRIPTION
**Pull request by hughhhh**
### SUMMARY

Add documentation to allow people understand how they can enable ssh tunneling and configure it for their own instance of superset.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by Antonio-RiveroMartnez — APPROVED** [review]
LGTM!

**Review by lilykuang — APPROVED** [review]

File: docs/docs/installation/setup-ssh-tunneling.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+title: Setup SSH Tunneling
+hide_title: true
+sidebar_position: 13
+version: 1
+---
+
+## SSH Tunneling
+
+1. Turn on feature flag
+    - Change (this value)[https://github.com/apache/superset/blob/master/superset/config.py#L489] to `True`
```
Comment (betodealmeida) [inline docs/docs/installation/setup-ssh-tunneling.mdx:11]: This link is going to be broken very quickly. You can use a link to a specific revision:

https://github.com/apache/superset/blob/eb8386e3f0647df6d1bbde8b42073850796cc16f/superset/config.py#L489

But even better would be also including what feature flag should be changed:

```suggestion
    - Change (`SSH_TUNNELING`)[https://github.com/apache/superset/blob/eb8386e3f0647df6d1bbde8b42073850796cc16f/superset/config.py#L489] to `True`
```

File: docs/docs/installation/setup-ssh-tunneling.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    - If you want to add more security when establish the tunnel we allow users to overwrite the `SSHTunnelManager` class (here)[https://github.com/apache/superset/blob/master/superset/config.py#L507]
```
Comment (betodealmeida) [inline docs/docs/installation/setup-ssh-tunneling.mdx:12]:
```suggestion
    - If you want to add more security when establishing the tunnel we allow users to overwrite the `SSHTunnelManager` class (here)[https://github.com/apache/superset/blob/master/superset/config.py#L507]
```

Same here, link to a specific SHA like I did above.

File: docs/docs/installation/setup-ssh-tunneling.mdx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    - You can also set the (`SSH_TUNNEL_LOCAL_BIND_ADDRESS`)[https://github.com/apache/superset/blob/master/superset/config.py#L508] this the host address where the tunnel will be accessible on your VPC
```
Comment (betodealmeida) [inline docs/docs/installation/setup-ssh-tunneling.mdx:13]: Ditto.

File: docs/docs/installation/setup-ssh-tunneling.mdx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+2. Create database w/ ssh tunnel enabled
+    - With the feature flag enabled you should now see ssh tunnel toggle
+    - Click the toggle to enables ssh tunneling and add your credentials accordingly.
+        - Superset allows for 2 different type authenticaion (Basic + Private Key). These credentials should come from your service provider.
+
+3. Verify data is flowing
+    - After goto SQL Lab, and write a query to verify data is properly flowing.
```
Comment (betodealmeida) [inline docs/docs/installation/setup-ssh-tunneling.mdx:21]:
```suggestion
    - Once SSH tunneling has been enabled, go to SQL Lab and write a query to verify data is properly flowing.
```

File: docs/docs/installation/setup-ssh-tunneling.mdx [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+## SSH Tunneling
+
+1. Turn on feature flag
+    - Change (this value)[https://github.com/apache/superset/blob/master/superset/config.py#L489] to `True`
+    - If you want to add more security when establish the tunnel we allow users to overwrite the `SSHTunnelManager` class (here)[https://github.com/apache/superset/blob/master/superset/config.py#L507]
+    - You can also set the (`SSH_TUNNEL_LOCAL_BIND_ADDRESS`)[https://github.com/apache/superset/blob/master/superset/config.py#L508] this the host address where the tunnel will be accessible on your VPC
+
+2. Create database w/ ssh tunnel enabled
+    - With the feature flag enabled you should now see ssh tunnel toggle
```
Comment (betodealmeida) [inline docs/docs/installation/setup-ssh-tunneling.mdx:16]:
```suggestion
    - With the feature flag enabled you should now see ssh tunnel toggle.
```

**Review by betodealmeida — CHANGES_REQUESTED** [review]

**Review by betodealmeida — APPROVED** [review]

**Pull request merged**